schemas/produktside/innholdsseksjon/InnholdsseksjonPreview.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { MdWeb } from "react-icons/md";
+import { PortableText } from "@portabletext/react";
+// @ts-ignore
+import styles from "./InnholdsseksjonPreview.module.css";
+
+export const InnholdsseksjonIkon = MdWeb;
+
+//@ts-ignore
+const InnholdsseksjonPreview = (props) => (
```

Comment:
Hvorfor ts-ignore her?
Vi har forøvrig brukt function for alle komponenter og generelt på øverste nivå. Vi bruker bare arrow-functions på maps og alle in-line type loops

Reply:
Kan godt se på det sammen evt om du har noen forslag her.

Kranglet mye med Typescript og Eslint da jeg startet på oppgaven. 
Jeg lagde først interface med Props også prøvde jeg å sende det inn som props via preview sin component-felt, men den likte ikke at jeg skrev JSX i component-feltet. Merkelig nok funker dette på alle de andre Previews (f.eks FremhevetTekst) der component er spesifisert og props er sendt inn.  Men mulig det er bare jeg misforstår noen ting, er relativt ny i Sanity og Typescript-oppsettet i denne repoen.